csss-site/src/events/tech_fair/static/tech_fair/2023/js/tech_fair_2023.js

```diff
@@ -136,7 +136,7 @@ function update(time) {
 }
 
 function updateBanner() {
-    document.getElementById("info-banner").hidden = false;
-    document.getElementById("info-banner").style.height = document.getElementById("info-text").clientHeight + 24 + 2 + "px";
+    document.getElementById("info-banner-bg").hidden = false;
```

Comment:
line 67 of this file is throwing an error
![Screenshot from 2023-09-16 12-18-40](https://github.com/CSSS/csss-site/assets/14242088/089707f6-0d96-4b8c-859c-536eafc7aa32)

Reply:
reproduced & fixed, thanks! A function was being called on the wrong page.